validator/client/iface/validator_client.go

```diff
@@ -62,7 +62,7 @@ func (b *BeaconCommitteeSelection) UnmarshalJSON(input []byte) error {
 	return nil
 }
 
-type ValidatorClient interface {
+type Coordinator interface {
```

Comment:
I'm not sold on the word coordinator, to me coordinator would also define when these are called as well. 
Most of these functions are actions for participating in consensus, what about `ConsensusToolkit` or `ConsensusAgent`

Reply:
I reverted this change. We should probably split the interface into a few smaller interfaces, each responsible for its own this. Mixing `GetDuties` with `EventStreamIsRunning` doesn't feel right.